### PR TITLE
open graph integration

### PR DESCRIPTION
**Pull request by nyrrad**
OG integration
- site name
- title
- description
- image
- url

For individual spice pages og info reflects that particular spice

### CODE REVIEW

File: system/application/views/header.php [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+
+    <meta property="og:site_name" content="Cinnamon Spices" />
+    <meta property="og:title" content="<?= ucfirst($type) ?>" />
+    <?php if (preg_match('|^/themes.*?|',$_SERVER['REQUEST_URI'])) { ?>
+        <meta property="og:description" content="Change the look and feel of Cinnamon with themes!" />
+    <?php } elseif (preg_match('|^/applets.*?|',$_SERVER['REQUEST_URI'])) { ?>
+        <meta property="og:description" content="Add applets to your Cinnamon panel!" />
+    <?php } elseif (preg_match('|^/desklets.*?|',$_SERVER['REQUEST_URI'])) { ?>
+        <meta property="og:description" content="Add desklets on top of your desktop wallpaper!" />
+    <?php } elseif (preg_match('|^/extensions.*?|',$_SERVER['REQUEST_URI'])) { ?>
```
Comment (MrElectroNick) [inline system/application/views/header.php]: @nyrrad You have $type variable - why query REQUEST_URI? :laughing:

File: system/application/views/welcome_page.php [hunk 2]
```diff
@@ -4,7 +4,16 @@
     <title>Cinnamon Spices</title>
     <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
     <link rel="shortcut icon" href="/favicon.ico"/>
-    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/style.css?<?= time() ?>">
+
+    <meta property="og:site_name" content="Cinnamon Spices" />
+    <meta property="og:title" content="Cinnamon Spices" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="http://cinnamon-spices.linuxmint.com" />
```
Comment (MrElectroNick) [inline system/application/views/welcome_page.php]: should be https

File: system/application/views/header.php (hunk 1 above)
Comment (nyrrad) [inline system/application/views/header.php]: I just used the existing if elseif system that was there. I'll switch it to type.

File: system/application/views/welcome_page.php (hunk 2 above)
Comment (nyrrad) [inline system/application/views/welcome_page.php]: Good catch.

**Pull request merged**